react/FilePath/styles.styl

```diff
@@ -0,0 +1,14 @@
+.c-file-path  
+  display block
+  color var(--coolGrey)
```

Comment:
should be semantic

Reply:
Unless I'm mistaken there doesn't seem to be a semantic var for that specific color. What should we call it? It's not used exclusively on file paths. textGreyLight?

Reply:
Il n'y a pas de specs dans Figma sur le sujet ?

Reply:
Ca devrait être un text.secondary ou quelque chose comme ça plutôt qu'un coolGrey. Ce qui ferait que le thème inverted serait supporté

Reply:
la seul ref que je trouve de déclaration/naming est ici https://github.com/cozy/cozy-ui/blob/93e83e10d30e226e4977cf96030f50e7e5050c03/stylus/settings/palette.styl#L59

Je vais voir sur Figma. Par contre ce grey n'est pas le main grey, il est utilisé en text ligth

Reply:
@Crash-- do you have some insight here, please?